spec/dummy/app/controllers/rescue_controller.rb

```diff
@@ -0,0 +1,34 @@
+class RescueController < ApplicationController
+  class CustomStatusError < StandardError; end
+  class CustomTitleError < StandardError; end
+  class MessageTrueError < StandardError; end
+  class MessageProcError < StandardError; end
+  class MetaProcError < StandardError; end
+  class LogFalseError < StandardError; end
+  class CustomHandlerError < StandardError; end
+  class SpecialPostError < StandardError; end
+
+  class CustomErrorHandler < GraphitiErrors::ExceptionHandler
+    def status_code(e)
+      302
+    end
+  end
+
+  rescue_from(CustomStatusError) { |e| render_graphiti_exception(e, status: 301) }
```

Comment:
Just to firm my understanding - I think this is no longer how it works right? That's my mental understanding but when I run the tests I get `undefined method `register_exception' for ActionController::Base:Class` so hard to check. I can't see `render_graphiti_exception` defined anymore, is this just WIP stuff? My understanding is it would be replaced with `register_exception`

Reply:
Yeah, WIP :)

Reply:
Fixed.